review/configuration.py

```diff
@@ -0,0 +1,36 @@
+'''
+Constants are stored here for use in other parts of the xBlock.
+'''
+
+# Needed for review.py #
+# Eventually, this should be part of the xBlock fields as a Boolean
+SHOW_PROBLEMS = set([
+    # This is here for testing purposes. Do not remove
+    'DillonX/DAD101x/3T2017',
+    # Anant's course
+    'course-v1:MITx+6.002.3x+2T2016',
+])
+SHOW_VERTICAL = set([
+])
+
+# Needed for get_review_ids.py #
+'''
+The mappings here are necessary to grab the review content from a review
+version of the course (a copy where problems are not graded and have
+unlimited attempts).
+When accessed, the key is the course the learner is currently interacting
+with and the value is the corresponding review course.
+'''
+REVIEW_COURSE_MAPPING = {
+    # Course used for testing. DO NOT REMOVE
+    'DillonX/DAD101x/3T2017': 'DillonX/DAD101xr/3T2017',
+    # Anant's course
+    'course-v1:MITx+6.002.3x+2T2016': 'MITx+6.002.3xr+2T2016',
```

Comment:
This 🚢 might have sailed. But the review course name is quite hard to distinguish from the original course name. Perhaps it wouldn't hurt to name them something like: `DillonX/DAD101x_review/3T2017`